README.md

```diff
@@ -1,265 +1,85 @@
-`$ export KUBE_FEATURE_GATES="PersistentLocalVolumes=true,VolumeScheduling=true,MountPropagation=true"`
-
-These features default to true in Kubernetes 1.10 and later and thus do not need to be set.
-
-Regardless of the Kubernetes version, a storage class with _volumeBindingMode_ set to "WaitForFirstConsumer" needs to be created. Eg:
-```
-   kind: StorageClass
-   apiVersion: storage.k8s.io/v1
-   metadata:
-     name: <local-storage-name>
-   provisioner: kubernetes.io/no-provisioner
-   volumeBindingMode: WaitForFirstConsumer
-```
-
-
-### Start Cloning Images
-
-From the [example manifests](./manifests/example) you copied earlier, the necessary file is:
-- target-pvc.yaml
-
-###### Edit target-pvc.yaml:
-1.  `k8s.io/CloneRequest:` The name of the PVC we copy the image from (including its namespace). For example: "source-ns/golden-pvc".
-2. add the name of the storage class which defines `volumeBindingMode` per above. Note, this is not required in Kubernetes 1.10 and later.
-
-### Deploy the API Object
-1. (Optional) Create the namespace where the target PVC will be deployed:
-
-   `$ kubectl create ns <TARGET-NAMESPACE>`
-
-1. Deploy the target PVC:
-
-   `$ kubectl -n <TARGET-NAMESPACE> create -f target-pvc.yaml`
-
-1. Monitor the cloning pods:
-
-   `$ kubectl -n <SOURCE-NAMESPACE> logs <clone-source-pod-name>`
-
-   `$ kubectl -n <TARGET-NAMESPACE> logs <clone-target-pod-name>`
-
-1. Check the target PVC for 'k8s.io/CloneOf' annotation:
-
-   `$ kubectl -n <TARGET-NAMESPACE> get pvc <target-pvc-name> -o yaml`
-
-
-### Security Configurations
-
-#### RBAC Roles
-
-CDI runs under a custom ServiceAccount (cdi-sa) and uses the [Kubernetes RBAC model](https://kubernetes.io/docs/admin/authorization/rbac/) to apply an application specific custom ClusterRole with rules to properly access needed resources such as PersistentVolumeClaims and Pods.
-
-
-#### Protecting VM Image Namespaces
-
-Currently there is no support for automatically implementing [Kubernetes ResourceQuotas](https://kubernetes.io/docs/concepts/policy/resource-quotas/) and Limits on desired namespaces and resources, therefore administrators need to manually lock down all new namespaces from being able to use the StorageClass associated with CDI/Kubevirt and cloning capabilities. This capability of automatically restricting resources is planned for future releases. Below are some examples of how one might achieve this level of resource protection:
-
-- Lock Down StorageClass Usage for Namespace:
+CDI includes a self contained development and test environment.  We use Docker to build, and we provide a simple way to get a test cluster up and running on your laptop.
 
 ```
-apiVersion: v1
-kind: ResourceQuota
-metadata:
-  name: protect-mynamespace
-spec:
-  hard:
-    <STORAGE-CLASS-NAME>.storageclass.storage.k8s.io/requests.storage: "0"
+$ git clone https://github.com/kubevirt/containerized-data-importer && cd containerized-data-importer
```

Comment:
Don't forget to mention the clone needs to happen in $GOPATH/src/kubevirt.io otherwise the cluster commands won't work since the compile will fail.

Reply:
Added it in the steps, thanks!